xonsh/main.py

```diff
@@ -148,17 +148,21 @@ def undo_args(args):
                     au[k](args)
 
 
+LEXER = pyghooks.XonshLexer()
```

Comment:
This might not be safe if pygments is not installed.  Maybe pyghooks itself would be a better place for the default lexer instance to live?